src/controllers/queriesController.js

```diff
@@ -0,0 +1,64 @@
+import {
+  createQuery,
+  getAllQueries,
+  getOneQuery,
+  deleteQuery,
+} from "../services/queryServices.js";
+
+export class QueryControllers {
+  async sendQuery(req, res, next) {
+    try {
+      const data = {
+        name: req.body.name,
+        email: req.body.email,
+        subject: req.body.subject,
+        message: req.body.message,
+        location: req.body.location,
+      };
+      const query = await createQuery(data);
+      res.status(201).json({
+        status: 201,
+        message: "Your querry is sent successfully",
+        data: query,
+      });
+    } catch (error) {
+      console.log(error);
```

Comment:
Return appropriate message and implement the same on other methods

Reply:
I am done